docs/reference/search/search-your-data/full-text-search.asciidoc

```diff
@@ -12,27 +12,29 @@ Full-text search, also known as lexical search, is a technique for fast, efficie
 Documents and search queries are transformed to enable returning https://www.elastic.co/what-is/search-relevance[relevant] results instead of simply exact term matches.
 Fields of type <<text-field-type,`text`>> are analyzed and indexed for full-text search.
 
-Built on decades of information retrieval research, full-text search in {es} is a compute-efficient, deterministic approach that scales predictably with data volume.
-Full-text search is the cornerstone of production-grade search solutions.
-Combine full-text search with <<semantic-search,semantic search using vectors>> to build modern hybrid search applications.
+Built on decades of information retrieval research, full-text search delivers reliable results that scale predictably as your data grows. Because it runs efficiently on CPUs, {es}'s full-text search requires minimal computational resources compared to GPU-intensive vector operations.
+
+This translates to lower infrastructure costs and predictable scaling requirements. You can scale horizontally by adding more nodes with standard CPU cores and RAM - no specialized hardware needed. A typical deployment will start with 2-3 nodes and grow incrementally as search volume increases. Learn more about <<scalability, moving to production>>.
```

Comment:
I think this is a bit of a red herring in this doc. I'd just remove the whole paragraph. it also sends the wrong signals to people on serverless who use ft search (the paragraph immediately before it also has references to hardware but I'm less concerned about it because it mostly just sells that this is a performant design)

when I mentioned prod in this context, I mostly meant the idea of making these calls from an app or site (this comment likely also a red herring)

Reply:
hmmm yes good point about serverless and the basic message is clear in preceding paragraph anyways